public/weboc-default-style.css

```diff
@@ -6,6 +6,7 @@
   --contrast-color: black;
   --canvas-fill: rgb(223, 223, 223);
   --weboc-app-bar-bg-color: #080c80;
+  --weboc-app-bar-bg-image: url('/images/blue_sky.jpg');
```

Comment:
don't set this as web-oc default. Will probably only be used for the metoc

Reply:
Should the image then even be uploaded to the public folder?

Reply:
Move to custom stylesheet